app/src/main/java/org/hackillinois/android/utils/TimeInfo.kt

```diff
@@ -0,0 +1,21 @@
+package org.hackillinois.android.utils
+
+class TimeInfo(val millis: Long) {
+    val days: Long
+    val hours: Long
+    val minutes: Long
+    val seconds: Long
+
+    init {
+        var leftOver = millis
+        days = leftOver / 86400000
```

Comment:
Make these numbers constants

Reply:
Also modular arithmetic might make this cleaner

Reply:
time /= MILLISECONDS_IN_SECOND

seconds = time % SECONDS_IN_MINUTE
time /= SECONDS_IN_MINUTE

minutes = time % MINUTES_IN_HOUR
time /= MINUTES_IN_HOUR

etc.